PyZAPI/pyzapi/views/notfound.py

```diff
@@ -0,0 +1,7 @@
+from pyramid.view import notfound_view_config
+
+
+@notfound_view_config(renderer='../templates/404.jinja2')
```

Comment:
Hi Mark, This was a default setup (created file) by pyramid, we can change it to a rest 404.